source/js/fellowships.js

```diff
@@ -5,8 +5,146 @@ import ReactDOM from 'react-dom';
 
 import Person from './components/people/person.jsx';
 
+let pulseApiDomain = ``;
+
+function getFellows(params, callback) {
+  Object.assign(params, {'profile_type': `fellow`});
+
+  let queryString = Object.entries(params).map(pair => pair.map(encodeURIComponent).join(`=`)).join(`&`);
+  let req = new XMLHttpRequest();
+
+  req.addEventListener(`load`, () => {
+    callback.call(this, JSON.parse(req.response));
+  });
+
+  req.open(`GET`, `https://${pulseApiDomain}/api/pulse/profiles/?${queryString}`);
+  req.send();
+}
+
+function renderFellowCard(fellow) {
```

Comment:
Can you put a comment in describing what this is doing (vs, say, the existing `person.jsx` component)